packages/tools/fluid-runner/src/test/exportFile.spec.ts

```diff
@@ -31,7 +31,7 @@ describe("exportFile", () => {
         describe(`Export using snapshot [${snapshotFileName}]`, () => {
             it("Output file is correct", async () => {
                 const exportFileResult = await exportFile(
-                    await fluidExport,
+                    fluidExport,
```

Comment:
@kian-thompson can you explain these removals of `await`?

Reply:
Oh never mind I see, `getFluidExport` was returning a Promise before.  I was worried it was more fundamental than that. (I was mixing this up with how you need to await require(...) in JS, but that's totally unrelated)